tap_tiktok_ads/client.py

```diff
@@ -7,15 +7,24 @@
 
 LOGGER = singer.get_logger()
 
+PROD_ENDPOINT_BASE = "https://business-api.tiktok.com/open_api/v1.2"
```

Comment:
Instead of defining 2 different Variables can we use format? like mentioned below: 
`"https://{api}.tiktok.com/open_api/v1.2".format(api="business-api")`